service/terraform-plugin/src/test/java/com/solace/maas/ep/event/management/agent/plugin/terraform/TerraformCommandIT.java

```diff
@@ -209,7 +217,7 @@ public void testImportResourceWithSomeResourcesExistingAndSomeNot() throws IOExc
                 "\tid = \"default/MyConsumer1/a%2Fb%2Fc\"\n" +
                 "}";
 
-        String content = Files.readString(Path.of(terraformProperties.getWorkingDirectoryRoot() + "/app123-ms1234/import.tf"));
+        String content = Files.readString(Path.of(terraformProperties.getWorkingDirectoryRoot() + "/app123-ms1234/sync.tf"));
```

Comment:
This test was passing for me locally since I still had the import.tf file in my local directory but it should have failed since we renamed the file to `sync.tf`. I changed the workingDirectoryRoot for tests to use `java.io.tmpdir` instead of `${HOME}/tfcommands` so it wouldn't just use the same directory every time the tests are run.